packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -93,6 +93,16 @@ class _LocalMicrophoneStream extends WcmeLocalMicrophoneStream {
       this[LocalMicrophoneStreamEventNames.ServerMuted].emit(muted, reason);
     }
   }
+
+  toJSON() {
```

Comment:
I think long term we should have toJSON in webrtc-core LocalStream class, but until that's done, we can do it here. I've only done if to mic and camera (not share), because web app only does effects on these 2 and this is a temporary fix.

Reply:
what exactly are we loggin , cant the logging go inside the SDK ?